src/poprox_concepts/domain/profile.py

```diff
@@ -0,0 +1,14 @@
+from typing import Dict, List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+from poprox_concepts.domain.click_history import ClickHistory
+from poprox_concepts.domain.account import AccountInterest
+
+
+class InterestProfile(BaseModel):
+    click_history: ClickHistory
+    click_topics: Optional[Dict[str, int]] = None
```

Comment:
The meaning of this is not immediately apparent. I'm _assuming_ it's a dictionary from topic to count of clicks on articles with that topic?

If we want to make it more apparent here we could either update to `topics_click_counts` or just add a quick comment `#click_topics[topic] is the count of clicks on articles with a given topic` or similar.